pyscf/solvent/pcm.py

```diff
@@ -0,0 +1,627 @@
+#!/usr/bin/env python
+# Copyright 2014-2023 The PySCF Developers. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Author: Xiaojie Wu <wxj6000@gmail.com>
+#
+
+'''
+PCM family solvent models
+'''
+
+import numpy
+import scipy
+import ctypes
+from pyscf import lib
+from pyscf.lib import logger
+from pyscf import gto, df
+from pyscf.dft import gen_grid
+from pyscf.data import radii
+from pyscf.solvent import ddcosmo
+from pyscf.solvent import _attach_solvent
+
+libdft = lib.load_library('libdft')
+
+@lib.with_doc(_attach_solvent._for_scf.__doc__)
+def pcm_for_scf(mf, solvent_obj=None, dm=None):
+    if solvent_obj is None:
+        solvent_obj = PCM(mf.mol)
+    return _attach_solvent._for_scf(mf, solvent_obj, dm)
+
+
+# Inject PCM to other methods
+from pyscf import scf
+from pyscf import mcscf
+from pyscf import mp, ci, cc
+from pyscf import tdscf
```

Comment:
These modules have not been used in this module. They should not be imported here.